[DOCS] Adds info about custom dashboards to dashboards landing page doc (#2533)
elastic/security-docs: docs/dashboards/dashboards-overview.asciidoc

The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.

> You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear at the bottom of the page:

Reviewer: ```suggestion
You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Once created, custom dashboards appear on the page:
```

Reviewer: Made some slight modifications based on our guidelines for writing for accessibility which recommend avoiding directional language. 
https://docs.elastic.dev/content-design/accessibility

(I sometimes forget about these guidelines so I'm sure I'll be giving myself the same edits soon ;) )

Reviewer 2: Accessibility is tricky. Tbh, sometimes I think it's necessary, especially in a complex or "noisy" UI, but understand why there are guidelines in there to suggest against directional language. You could also say something like `"Once created, confirm that your custom dashboard appears on the page." `

> You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear under CUSTOM at the bottom of the page:

Reviewer 3: All-caps CUSTOM feels shouty -- I know our [style guide](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#general-guidelines-for-writing-about-the-ui) says to reproduce the capitalization from the UI, but I think that's for Title Case or Sentence case, not all-caps. Maybe change to "Custom" (and bold?), or consider omitting entirely -- we may not need to name the section, just say where it is:
```suggestion
You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear at the bottom of the page:
```
